### PR TITLE
Use external align-to-origin util

### LINKED ISSUE #1183: Improve origin adjustment feature
In order to limit the effect of this, could we also update the default template for new files to place the start event at x="140" y="188". Why that position? The x coordinate allows a start event with a maximally wide label to be visible when a model is opened and the palette uses two columns. The y coordinate allows using a standard-size embedded sub-process behind the start event without triggering the behavior implemented for this issue.

20190209 camunda modeler 001

In fact the x="140" should be the destination to move to, because otherwhise the two-column palette on small screens covers the start event or its label.

20190209 selection 002
20190209 selection 003

Furthermore, I think the 'Move to origin' should be aligned with this, because otherwise users that used 'Move to origin' will have a hard time moving their diagrams to a position where its start event is not hidden behind the palette when they open the file. Basically, their only chance would be to use the space tool or create new elements in negative areas, to trigger the behavior implemented for this issue.

_Originally posted by @falko in https://github.com/camunda/camunda-modeler/issues/982#issuecomment-462052615_

### PR DESCRIPTION
**Pull request by nikku**
This is a follow up to https://github.com/camunda/camunda-modeler/pull/1170.

* use external align-to-origin util
* disable move-to-origin feature if automatic alignment is configured

Closes #1183

### CODE REVIEW

**Comment by nikku** [comment]
Ready for review :nerd_face:

**Comment by nikku** [comment]
Closes #1183

File: client/src/app/tabs/dmn/diagram.dmn [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<definitions xmlns="http://www.omg.org/spec/DMN/20151101/dmn.xsd" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/1.0" id="Definitions_{{ ID }}" name="DRD" namespace="http://camunda.org/schema/1.0/dmn">
```
Comment (pinussilvestrus) [inline client/src/app/tabs/dmn/diagram.dmn]: Any reason why we remove the `{{ ID }}` placeholder, which got filled [later on](https://github.com/camunda/camunda-modeler/blob/master/client/src/app/TabsProvider.js#L217)?

Comment (nikku) [reply client/src/app/tabs/dmn/diagram.dmn]: Restored the placeholder + added tests via https://github.com/camunda/camunda-modeler/pull/1186/commits/3335b042735c6e7645e05d38bd5e4a72155497fc.

**Comment by nikku** [comment]
Ready for second review round :racehorse:.

File: client/src/app/tabs/bpmn/__tests__/DiagramSpec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+describe('tabs/bpmn', function() {
```
Comment (pinussilvestrus) [inline client/src/app/tabs/bpmn/__tests__/DiagramSpec.js]: ❤

**Pull request merged**